map_test.go

```diff
@@ -14,9 +14,13 @@ func TestKeys(t *testing.T) {
 	is := assert.New(t)
 
 	r1 := Keys(map[string]int{"foo": 1, "bar": 2})
-	sort.Strings(r1)
```

Comment:
You must keep the sort, or add it to Keys function, because a map is unordered while a slice is. So the way the loop is made on the map is random, so the order in the slide is random

Reply:
Got it. My bad this PR made me learn a lot. That is why we had sort in the first place in the test functions.
Sure I have added it back.
And as for sorting in the Keys() function, let's not add it. It will just add more time complexity to the function. If any user wants to get it sorted. They can sort it after the slice is returned (depending on there usecase)

What are your thoughts ??